web-console/pom.xml

```diff
@@ -32,13 +32,27 @@
   </parent>
 
   <properties>
+    <maven.build.cache.input.1>src</maven.build.cache.input.1>
+    <maven.build.cache.input.2>public</maven.build.cache.input.2>
```

Comment:
you do not need to include `public`, it is 100% generated from the other things on build. The rest look 💯

Reply:
I've removed it!
it was even in `.gitignore` - didn't notice that! :)